contracts/token/ERC20.cairo

```diff
@@ -12,6 +12,14 @@ from starkware.cairo.common.uint256 import (
 # Storage
 #
 
+@storage_var
+func _name() -> (res: felt):
```

Comment:
I think we can drop the underscore right? There's no collision risk.

Reply:
The compiler doesn't like it. "Redefinition of name" because it's in `view` and `storage_var`